tests/test-email-prepare.php

```diff
@@ -0,0 +1,94 @@
+<?php
+include_once dirname( __FILE__ ) . '/email-test-case.php';
+
+/**
+ * Class testing Caldera_Forms_Email_Prepare
+ */
+class Test_Email_Prepared extends Email_Test_Case {
+
+
+	/**
+	 * test return email from rfc822
+	 */
+	public function test_email_from_rfc822() {
+		$_rfc      = 'Roy Sivan <roy@sivan.com>';
+		$_expected = 'roy@sivan.com';
+
+		$_actual = Caldera_Forms_Email_Prepare::email_from_rfc822( $_rfc );
+
+		$this->assertEquals( $_expected, $_actual );
+	}
+
+
+	/**
+	 * test return name from rfc822
+	 */
+	public function test_name_from_rfc822() {
+		$_rfc      = 'Roy Sivan <roy@sivan.com>';
+		$_expected = 'Roy Sivan';
+
+		$_actual = Caldera_Forms_Email_Prepare::name_from_rfc822( $_rfc );
+
+		$this->assertEquals( $_expected, $_actual );
+	}
+
+
+	/**
+	 * check the email is rfc_822 format
+	 */
+	public function test_is_rfc822() {
+		$_rfc    = 'Roy Sivan <roy@sivan.com>';
+		$_is_rfc = Caldera_Forms_Email_Prepare::is_rfc822( $_rfc );
+
+		$this->assertNotNull( $_is_rfc );
+	}
+
+
+	/**
+	 * prepare array of emails and names using the dataProvider
+	 *
+	 * @dataProvider preparedEmailFormats
+	 *
+	 * @param $original
```

Comment:
@robwilde Method parameters need type  and descriptions. 
http://docs.phpdoc.org/references/phpdoc/tags/param.html